spec/services/csv/append_notes_content_spec.rb

```diff
@@ -0,0 +1,54 @@
+require "spec_helper"
+
+describe Csv::AppendNotesContent do
+  let(:csv) { [] }
+  let(:date) { DateTime.new(2018,8,20,11,16,44) }
+  let(:latitude) { BigDecimal.new("40.68038924") }
+  let(:longitude) { BigDecimal.new("-73.97631499") }
+  let(:note1) { { text: "Example Note", date: date, latitude: latitude, longitude: longitude, } }
+  let(:note_with_comma) { { text: 'Example, Note', date: date, latitude: latitude, longitude: longitude, } }
+  let(:note2) { { text: "Example Note 2", date: date, latitude: latitude, longitude: longitude, } }
+  let(:expected_headers) { %w(Note Time Latitude Longitude) }
```

Comment:
For me, it's the opposite. It's easier to read when the `its` are shorter and the setup is in the `lets`.
Also, I would be careful about going against common conventions. The big advantage of conventions is that it's easier for people to join new projects. And I think using `lets` is a very popular technic in rspec. 
It might be easier for someone to read something they invented and use for years, but if everybody did that it would be extremely hard to collaborate with each other. So unless it's a very harmful technic, it's better to stick to convention imo
 
As for `lets` that are not used in all `its`, I didn't realize it's a problem. In a similar fashion, we extract methods in ruby. Even if a method is only used in one place usually it's a good idea to extract. In the future, someone will be adding something to this code and they will notice the method and use it.

Reply:
I don't agree but I'm fine with that and appreciate you having a different opinion!